src/layout/VerifyEmail.component.js

```diff
@@ -37,23 +34,21 @@ export function VerifyEmail({ userEmail }) {
 
     const emailConfigured = systemInfo?.emailConfigured
 
-    const isValidEmail = emailRegExp.test(userEmail)
+    const isInvalidEmail = Boolean(emailValidator(userEmail))
```

Comment:
this is just updating the validator to be consistent with what we have in the email modal